source/triggers/aws-eventbridge.txt

```diff
@@ -152,6 +168,233 @@ Procedure
 
+.. _eventbridge-error-handling:
+
+Custom Error Handling
+---------------------
+
+.. note:: Only Database Triggers Support Custom Error Handlers
+
+   Currently, only database triggers support custom error handling. 
+   Authentication triggers and scheduled triggers do not support 
+   custom error handling at this time. 
+
+.. figure:: /images/eventbridge-error-handling-diagram.png
+   :alt: EventBridge Custom Error Handling Flow Diagram.
+
+You can create an error handler to be executed on a trigger failure, 
+when retry does not succeed. Custom error handling allows you to determine
+whether an error from AWS EventBridge is critical enough to suspend the Trigger,
+or if it is acceptable to ignore the error and continue processing other events.
+For more information on suspended databse triggers, refer to 
+:ref:`Suspended Triggers <suspended_triggers>`.
+
+Create a New Custom Error Handler
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: ui
+
+      .. figure:: /images/eventbridge-custom-function.png
+         :alt: The EventBridge custom error handling configuration in the UI.
+
+      .. procedure::
+
+         .. step:: Create a New Error Handler
+
+            In the :guilabel:`Configure Error Function` section, select 
+            :guilabel:`+ New Function`.
+
+            You can also select an existing Function, if one is already defined,  
+            from the dropdown. For more information, see 
+            :ref:`Define a Function <define-a-function>`.
+
+         .. step:: Name the New Function
+
+            Enter a unique, identifying name for the function in the :guilabel:`Name` field. 
+            This name must be distinct from all other functions in the application.
+
+         .. step:: Write the Function Code
+
+            In the :guilabel:`Function` section, write the function in the function editor.
+            In the function editor, you will see a default function. You can edit
+            the function as needed. 
+
+         .. step:: Test the Function
+
+            In the :guilabel:`Testing Console` tab beneath the function editor, you can
+            test the function by passing in example values to the ``error`` and 
+            ``changeEvent`` parameters, as shown in the comments of the testing console. 
+
+            For more information on these paramaters, refer to the 
+            :ref:`Error Handler Parameters <eventbridge-error-handler-parameters>`
+            section on this page.
+
+            Click :guilabel:`Run` to run the test. 
+
+         .. step:: Save the Function
+
+            Once you are satisfied with the custom error handler, click
+            :guilabel:`Save`. 
+
+   .. tab::
+      :tabid: cli
+
+      In order to update your trigger's configuration with an error handler, 
```

Comment:
I really don't think this is necessary. if we are providing this info, then it seems more appropriate to be in the above section's CLI procedure (step 2), not here

Reply:
I like the structure as is. For users wanting to add an error handler to a pre-existing trigger, separating the procedures makes the most sense for them, so they don't have to read through the configure a new trigger material. Did slight reword to make following procedure less clunky.